### ISSUE TITLE
Profile page for users who have not submitted a game shows all games

### ISSUE THREAD

**Issue by balohmatevz** [OneHourGameJam/OneHourGameJam#85]
Example: https://onehourgamejam.com/?page=author&author=limetest5

![image](https://user-images.githubusercontent.com/1364475/50571709-2ac99980-0da9-11e9-97af-8707ae486d62.png)

**Comment by LongarMD** [comment]
Is this still a problem? I can't seem to replicate it. In fact, it's quite the opposite (see picture).
![image](https://user-images.githubusercontent.com/21366959/50995300-05622b80-151f-11e9-94dd-960541718219.png)

**Comment by balohmatevz** [comment]
Yes, what you posted is the new state of this error, it's still the case
